Textmining/src/org/wisslab/ss15/textmining/Speaker.java

```diff
@@ -70,22 +70,22 @@ public double getTFIDF(String word) {
         int D = work.getSpeakers().size();
         return tf * Math.log(D/df);
     }
-    
+
     public List<Double> getVSMVector() {
         List<Double> result = new ArrayList<>();
         for (String word: work.getDF().keySet()) {
             result.add(getTFIDF(word));
         }
         return result;
     }
-    
+
     public Map<String, Double> getTF() {
         Map<String, Double> result = new HashMap<>();
         Pattern p1 = Pattern.compile("[a-zA-Z']+");
-     
+
         for (String word: getAllText().split(" ")) {
             // Das ist quasi Tokenization, dafür gibt es Spezialbibliotheken
-            word = word.replaceAll(",", "");
+            word = word.replaceAll("\\,", "");
```

Comment:
Ne, das Komma warvok. Nur der Punkt musste escaped werdeb. Ich habe das wohl in dem Form falsch geändert. 
Ich habe das neue überarbeitet Programm angeschaut. Ja, es ist tatsächlich schneller und liefert viele interessante Ergebnisse. Aber um das zu verstehen, muss ich mich mal einen Tag da dran setzten und alles durchgehen.